synthtool/gcp/templates/python_notebooks_testing_pipeline/.cloud-build/cleanup/cleanup.py

```diff
@@ -36,15 +37,18 @@ def run_cleanup_managers(managers: List[ResourceCleanupManager], is_dry_run: boo
                 resource_name = manager.resource_name(resource)
                 print(f"Will delete '{type_name}': {resource_name}")
             else:
-                manager.delete(resource)
+                try:
+                    manager.delete(resource)
+                except Exception as exception:
+                    print(exception)
 
+        print("")
 
-def set_dry_run(dry_run_status: bool):
-    if dry_run_status is True:
-        return True
-    print("Starting cleanup in dry run mode...")
-    return False
 
+is_dry_run = False
+
+if is_dry_run:
+    print("Starting cleanup in dry run mode...")
```

Comment:
`is_dry_run` is always False, please remove test code or add a comment to clarify the reason for having it.

Reply:
Removed - I think the setting fixes an immediate problem but the print statement exists for future settings of that variable. However, I tend to not include code until it's supported. @ivanmkc may have more context!